### PR TITLE
first task

### PR DESCRIPTION
**Pull request by Alexandr98**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (LimarenkoDenis) [inline src/index.html]: delete html and run script using node js

File: src/loops/checkAge.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export function checkAge(age) {
+  if (age > 18) {
+    return true;
+  } else {
```
Comment (LimarenkoDenis) [inline src/loops/checkAge.js]: delete else construction

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+root = true
```
Comment (LimarenkoDenis) [inline .editorconfig]: this file already exists with this changes

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
```
Comment (LimarenkoDenis) [inline .eslintrc.js]: this file already exists with this changes

File: package.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (LimarenkoDenis) [inline package.json]: this file already exists with this changes

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,2 @@
+/node_modules
```
Comment (LimarenkoDenis) [inline .gitignore]: this file already exists with this changes

File: package.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  "name": "learning",
+  "version": "1.0.0",
+  "description": "learning js",
+  "main": "index.js",
+  "dependencies": {
+    "eslint": "^4.19.0",
```
Comment (NepipenkoIgor) [inline package.json]: fix version

File: src/function/pow.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+export function pow(x, n) {
+  let res = x;
+  for (let i = 0; i < n; i++) {
+    res = res * x;
```
Comment (NepipenkoIgor) [inline src/function/pow.js]: what about NaN

File: src/loops/checkAge.js (hunk 2 above)
Comment (NepipenkoIgor) [inline src/loops/checkAge.js]: why you use  else ?

File: src/loops/evenNumbers.js [hunk 9]
```diff
@@ -0,0 +1,10 @@
+export function evenNum(i, a) {
+  let arr = [];
+  for (i = 0; i < a; i++) {
+    if (i % 2 === 0) {
+      arr.push(i);
```
Comment (NepipenkoIgor) [inline src/loops/evenNumbers.js]: inverse logic and use continue

File: src/loops/repeatLoop.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
+let value;
```
Comment (LimarenkoDenis) [inline src/loops/repeatLoop.js]: write pure function, without global variables, and let's get a value as an argument(just add second argument), not from prompt

**Comment by LimarenkoDenis** [comment]
@Alexandr98 fix conflicts and take a look at our comments

File: src/function/pow.js [hunk 11]
```diff
@@ -0,0 +1,12 @@
+export function pow(x, n) {
+  let res = x;
+  if (isNan(x)) {
```
Comment (LimarenkoDenis) [inline src/function/pow.js]: let's use Number.isNaN

File: src/function/pow.js [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+    alert("NaN");
```
Comment (LimarenkoDenis) [inline src/function/pow.js]: let's return a default value, not alert

File: src/function/pow.js [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+  } else {
```
Comment (LimarenkoDenis) [inline src/function/pow.js]: see previous comments, delete else and use return

File: src/loops/evenNumbers.js [hunk 14]
```diff
@@ -0,0 +1,11 @@
+export function evenNum(i, a) {
+  let arr = [];
+  for (i = 0; i < a; i++) {
+    if (i % 2 !== 0) continue;
```
Comment (LimarenkoDenis) [inline src/loops/evenNumbers.js]: what's happening here, align code and use `{}` for if constructions

**Pull request closed without merging**